pkg/cfaws/session_name.go

```diff
@@ -0,0 +1,10 @@
+package cfaws
+
+import "github.com/segmentio/ksuid"
+
+// sessionName returns a unique session identifier for the aws console
+// this ensures that user activity can be easily audited per session
+// this uses the convenient ksuid library for generating unique IDs
+func sessionName() string {
+	return "gran-" + ksuid.New().String()
```

Comment:
Should this be "granted"?

Reply:
I needed the name to come in under a character limit, maybe I should just use a different UUID format.
I think I will swap out the KSUID for something else